gateway/1.4/modules/ROOT/pages/flex-gateway-k8-ingress-class.adoc

```diff
@@ -0,0 +1,61 @@
+= Configure Flex Gateway as an Ingress Controller in Local Mode
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+For Kubernetes deployments in Local Mode, Flex Gateway typically acts as an ingress controller that manages external access to your Kubernetes cluster.
+
+To configure Flex Gateway as an ingress controller, you must configure an `IngressClass` resource and an `Ingress` resource. The `IngressClass` resource configures Flex Gateway as an ingress controller, and the `Ingress` resources configures the services Flex Gateway manages as an ingress controller.
+
+By default, Flex Gateways deployed on Kubernetes in Local Mode are defined as `IngressClass` resources.
```

Comment:
`Flex gateway configures itself as an IngressClass by default`. Maybe merge this paragraph with the one above.